packages/dashboard/src/components/SubwayNav/SubwayNav.tsx

```diff
@@ -0,0 +1,298 @@
+import * as React from 'react';
+
+import { classNamesFunction, css } from 'office-ui-fabric-react/lib/Utilities';
+import { ISubwayNavProps, ISubwayNavStep, ISubwayNavStyles, SubwayNavStepState } from './SubwayNav.types';
+import { Icon, Link } from 'office-ui-fabric-react';
+import { getSubwayNavStyles } from './SubwayNav.styles';
+
+export interface IIconProps {
+  iconName: string;
+
+  iconClassName: string;
+}
+
+const getClassNames = classNamesFunction<ISubwayNavProps, ISubwayNavStyles>();
+const classNames = getClassNames(getSubwayNavStyles!);
+
+/**
+ * Component for Subway control
+ */
+export class SubwayNav extends React.Component<ISubwayNavProps, {}> {
+  constructor(props: ISubwayNavProps) {
+    super(props);
+  }
+
+  /**
+   * Render the Subway Nav control
+   */
+  public render(): JSX.Element {
+    const { steps /*theme, */ } = this.props;
+    let stepElements: JSX.Element[] = [];
+    let renderedSteps: JSX.Element[] = [];
+
+    if (steps) {
+      for (let stepIndex = 0; stepIndex < steps.length; stepIndex++) {
```

Comment:
A `map` seems better suited here, and performs better too.

Reply:
Fixed